common/src/Microsoft.Azure.IIoT.Auth.ActiveDirectory/src/Extensions/ConfigurationEx.cs

```diff
@@ -31,9 +31,10 @@ public static class ConfigurationEx {
         /// <param name="keyVaultUrlVarName"></param>
         /// <returns></returns>
         public static IConfigurationBuilder AddFromKeyVault(
-            this IConfigurationBuilder builder, bool singleton = true,
-            string keyVaultUrlVarName = null) {
-
+            this IConfigurationBuilder builder,
```

Comment:
Only change seems to be formatting, or?

Reply:
Yes, all changes to `ConfigurationEx.cs` are just cosmetic. I can revert those if you want.